src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt

```diff
@@ -16,6 +16,10 @@ import java.time.LocalDateTime
 @Transactional
 class PrisonSignedOperationalCapacityRepositoryTest : TestBase() {
 
+  val mdi = "MDI"
+  val xxx = "XXX"
```

Comment:
name of a variable called `xxx` is unclear

Reply:
testPrisonId **OK**, xxx was typo/left behind. corrected thanks